cmd/web-handlers.go

```diff
@@ -799,6 +856,9 @@ next:
 					UserAgent:  r.UserAgent(),
 					Host:       handlers.GetSourceIP(r),
 				})
+				go func() {
+					replicateDelete(GlobalContext, objectAPI, args.BucketName, deletedObjects, false)
+				}()
 			}
 		}
 	}
```

Comment:
So there is no control over how many delete calls will be called? - we shouldn't do this again - follow the same logic used for replicateObject() where we should control the calls - we cannot have replication take run-amok resources on the server. 

Client calls shouldn't be able to abuse server by exploting functionality causing a denial of service like attacks.

Reply:
updated the PR to queue deletes